src/main/resources/modules/cystic_fibrosis.json

```diff
@@ -0,0 +1,1649 @@
+                "since": "Develop_Infection_1",
+                "within": {
+                  "quantity": 6,
+                  "unit": "months"
+                }
+              }
+            ]
+          },
+          "distributions": [
+            {
+              "transition": "Sepsis_Of_Paeruginosa",
+              "distribution": 0.4
+            },
+            {
+              "transition": "patient_lives",
+              "distribution": 0.6
+            }
+          ]
+        },
+        {
+          "condition": {
+            "condition_type": "And",
+            "conditions": [
+              {
+                "condition_type": "PriorState",
+                "name": "Develop_Infection_2"
+              },
+              {
+                "condition_type": "PriorState",
+                "name": "Prescribe_Vancomycin",
+                "since": "Develop_Infection_2",
+                "within": {
+                  "quantity": 6,
+                  "unit": "months"
+                }
+              }
+            ]
+          },
+          "distributions": [
+            {
+              "transition": "Sepsis_of_Staphylococcus_aureus",
+              "distribution": 0.4
+            },
+            {
+              "transition": "patient_lives",
+              "distribution": 0.6
+            }
+          ]
+        },
+        {
+          "condition": {
+            "condition_type": "PriorState",
+            "name": "Develop_Infection_1"
+          },
+          "distributions": [
+            {
+              "distribution": 0.8,
+              "transition": "Sepsis_Of_Paeruginosa"
+            },
+            {
+              "transition": "pulmonary_exacerbation",
+              "distribution": 0.2
+            }
+          ]
+        },
+        {
+          "condition": {
+            "condition_type": "PriorState",
+            "name": "Develop_Infection_2"
+          },
+          "distributions": [
+            {
+              "distribution": 0.8,
+              "transition": "Sepsis_of_Staphylococcus_aureus"
+            },
+            {
+              "transition": "pulmonary_exacerbation",
+              "distribution": 0.2
+            }
+          ]
+        },
+        {
+          "distributions": [
+            {
+              "distribution": 1,
+              "transition": "pulmonary_exacerbation"
+            }
+          ]
+        }
+      ],
+      "remarks": [
+        "https://www.sciencedirect.com/science/article/pii/S1569199315002106?viewFullText=true#f0015"
+      ]
+    },
+    "Sepsis_Of_Paeruginosa": {
+      "type": "ConditionOnset",
+      "assign_to_attribute": "CF_Complication",
+      "target_encounter": "Emergency_Room",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 448813005,
+          "display": "Sepsis caused by Pseudomonas (disorder)"
+        }
+      ],
+      "direct_transition": "low_blood_pressure"
+    },
+    "Emergency_Room": {
```

Comment:
Every quarterly visit is an emergency room visit?

Reply:
Or at least, the two are mutually exclusive?

Reply:
I'm not sure what you mean here, the Quarterly Visit is ambulatory.

Reply:
Does a patient have an emergency room visit XOR a quarterly visit? Can they have both in the same quarter?

Reply:
They can't have both in the same quarter, they have to either get their checkup or go to the emergency room.

Reply:
OK, can you change that? If you have a standing quarterly visit, you're going to go to that, even if you happen to randomly and periodically end up the emergency room.

Reply:
Can do

Reply:
In the latest version if the patient goes to the emergency room and doesn't die they will have their quarterly checkup immediately after. This may be a good candidate to merge into 2 modules as there are parallel tracks here. At minimum there should be some delay between the end of the emergency room encounter and the start of the quarterly checkup.

Reply:
I added a delay state for now to fix this.  I don't know what you mean by "merge into 2 modules" though.

Reply:
Bad phrasing on my part. I meant split into 2 modules. It's possible for one module to represent multiple parallel tracks, but the logic can get complicated. Splitting it apart into multiple modules could make each one simpler, at the cost of having more modules to keep track of.

Yet another option would be to reduce the time delay in the loop so that it runs more often, for example every month, checking if the quarterly checkup hasn't happened in some time using a `PriorState` logic, then falling back to some percent chance of ER visit, or finally just looping back to the delay.

I'm ok with the current impl but I'll defer to @jawalonoski